jetty-ee10/jetty-ee10-annotations/src/test/java/org/eclipse/jetty/ee10/annotations/TestAnnotationInheritance.java

```diff
@@ -13,6 +13,12 @@
 
```

Comment:
I'm not sure this test has a lot of validity any more, as you removed the method that was walking the class hierarchy of a given class. All these tests are now copying in multiple classes to a directory, then passing that directory to the parser, so it's really now testing the `parseDir` method. Same comment for ee9.

Reply:
From commit: [Deleting TestAnnotationInheritance tests](https://github.com/eclipse/jetty.project/pull/8404/commits/f93e2268c3a2194fa10543f29697f69989e0a70e) 

+ We don't want the `AnnotationParser`
  creating new Resource objects for `Class.getSuperClass()` discovered classes
  resources that don't belong to the same `Resource` that the passed
  in class belongs to (as this is out of scope for the Mount management).

  Imagine if told to parse `ClassB`
  which extends from `ClassFoo`, which
  is in a different JAR than `ClassB`?

  This concept from Jetty 9/10/11 was only used in testcases and not
  in live runtime code, so it was  removed.